examples/cell/index.html

```diff
@@ -2,11 +2,11 @@
 <html lang="en">
   <head>
     <title>Cell Demo</title>
-    <script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_CHTML-full,Safe&amp;delayStartupUntil=configured"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS_CHTML-full,Safe&amp;delayStartupUntil=configured"></script>
```

Comment:
It looks like 2.7.7 is the latest 2.x release. Might as well move to the latest, if not to 3.0?

Reply:
OK to bump to 2.7.7 :+1: 

For 3.0, maybe it could be done as part of https://github.com/jupyterlab/jupyterlab/issues/7218?